Dockerfile.alpine

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.13-alpine
+
+# don't place it into $GOPATH/bin because Drone mounts $GOPATH as volume
+COPY golangci-lint /usr/bin/
+CMD ["golangci-lint"]
```

Comment:
did you try this image on `cgo` project? I guess it will fail without `gcc` tools installed

Reply:
I haven't performed any rigorous testing. Is there an easy way to test `cgo` support?

Reply:
yep, see `test/testdata/cgo*`

Reply:
I tried the `cgo` testdata and it seems to be working as expected after I added the `gcc` and `musl-dev` dependencies to the image.

see: https://github.com/golangci/golangci-lint/pull/745/files#diff-39bd4a9d0958b0d526ad2585d0738b26R4